default.json

```
@@ -103,6 +103,16 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the README.md on renovatebot/github-action repository",
+      "fileMatch": ["^README\\.md$"],
```

```suggestion
      "fileMatch": ["\\.md$"],
```

Update all markdown files